Lilypad-Tech/lilypad
# feat: Configuration update for Arbitrum Local l2

### Review Type Requested (choose one):
- [X ] **Logic** - thorough check (from everybody doing review)

### Summary
Added function to stack to deploy Arbitrum L2 in standalone mode
### How to test this code? (optional)
1. ./stack dev_deploy
2. followed instruction on screen to save envs
3. ./stack solver
You should see solver full initialized, waiting for Resource Providers and Jobs
